packages/lodestar/src/chain/validation/aggregateAndProof.ts

```diff
@@ -14,7 +14,8 @@ import {getCommitteeIndices, verifyHeadBlockAndTargetRoot, verifyPropagationSlot
 
 export async function validateGossipAggregateAndProof(
   chain: IBeaconChain,
-  signedAggregateAndProof: phase0.SignedAggregateAndProof
+  signedAggregateAndProof: phase0.SignedAggregateAndProof,
+  validateKnownAttesters: boolean = true,
```

Comment:
I think it's best to have a default that's the falsy value. For example use as variable: `skipValidationKnownAttesters`